openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -74,23 +74,24 @@ Note that when OpenID4VP is used, the Wallet and the Verifier can either be remo
 
 Assumptions made are the following:
 
-* The issuers and verifiers cannot pre-discover Wallet’s capability
-* The issuer is talking to the Wallet supporting the features defined in this profile (via Wallet invocation mechanism)
-* There are mechanisms in place for the verifiers and issuers to discover each other’s capability
+* The Issuers and Verifiers cannot pre-discover Wallet’s capability
+* The Issuer is talking to the Wallet supporting the features defined in this profile (via Wallet invocation mechanism)
+* There are mechanisms in place for the Verifiers and Issuers to discover each other’s capability
```

Comment:
what does this mean? Why need verifiers need to discover issuer capabilities and vice versa?